js/state/siteUtil.js

```diff
@@ -105,9 +106,14 @@ const mergeSiteDetails = (oldSiteDetail, newSiteDetail, tag, folderId) => {
   let site = Immutable.fromJS({
     lastAccessedTime: lastAccessedTime,
     tags,
-    title: newSiteDetail.get('title')
+    title: newSiteDetail.get('title'),
+    order
   })
 
+  if (oldSiteDetail && oldSiteDetail.get('order') !== undefined) {
+    site = site.set('order', oldSiteDetail.get('order'))
```

Comment:
At some point (doesn't have to be now, something for all of us to keep in mind :smile:), I think it might be worth creating util methods to help with type conversion / default values.  Something like this:

```
const safeNumber = (value, default) => {
  const num = Number(value)
  return isNaN(num) ? default : num
}
```